Booktrix/Extensions/UIViewController+SwiftNotifications.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  UIViewController+SwiftNotifications.swift
+//  Booktrix
+//
+//  Created by Impresyjna on 18.05.2017.
+//  Copyright © 2017 Impresyjna. All rights reserved.
+//
+
+import UIKit
+import BRYXBanner
+
+enum NotificationBackground : UInt32 {
+    case warning = 0xFFA000ff
+    case error = 0xD32F2Fff
+    case success = 0x388E3Cff
+}
```

Comment:
To jest dosyć nieczytelne. Może lepiej by to domyślnie zwracało kolor a nie jakiegoś hexa. I wtedy zrób extension na UIColor z `init(hex: )`. Dzięki temu kod jest bardziej deklaratywny bo operujesz na kolorze a nie na jego reprezentacji hexadecymalnej